diff --git a/dist/index.cjs b/dist/index.cjs
index 6f5d3667d5ea4282eed8e67d66f1bdd8580cae0b..c66e4584b7947c14934f7f8fc2151b9ef04701c0 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -14079,7 +14079,7 @@ class FontWrapper {
       this.relationships.addRelationship(
         i + 1,
         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/font",
-        `fonts/${options[i].name}.odttf`
+        `fonts/font${i + 1}.odttf`
       );
     }
   }
@@ -21599,9 +21599,8 @@ class Compiler {
         zip.file(`word/media/${data.fallback.fileName}`, data.fallback.data);
       }
     }
-    for (const { data: buffer2, name, fontKey } of file.FontTable.fontOptionsWithKey) {
-      const [nameWithoutExtension] = name.split(".");
-      zip.file(`word/fonts/${nameWithoutExtension}.odttf`, obfuscate(buffer2, fontKey));
+    for (const [i, { data: buffer2, fontKey }] of file.FontTable.fontOptionsWithKey.entries()) {
+      zip.file(`word/fonts/font${i + 1}.odttf`, obfuscate(buffer2, fontKey));
     }
     return zip;
   }
diff --git a/dist/index.mjs b/dist/index.mjs
index 82a8ca2c6ea44f603f4f8e1b69f0d7c5f00783c9..0c5941d32d90ab903ed19ae6c2027f7f4f33dfa7 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -14077,7 +14077,7 @@ class FontWrapper {
       this.relationships.addRelationship(
         i + 1,
         "http://schemas.openxmlformats.org/officeDocument/2006/relationships/font",
-        `fonts/${options[i].name}.odttf`
+        `fonts/font${i + 1}.odttf`
       );
     }
   }
@@ -21597,9 +21597,8 @@ class Compiler {
         zip.file(`word/media/${data.fallback.fileName}`, data.fallback.data);
       }
     }
-    for (const { data: buffer2, name, fontKey } of file.FontTable.fontOptionsWithKey) {
-      const [nameWithoutExtension] = name.split(".");
-      zip.file(`word/fonts/${nameWithoutExtension}.odttf`, obfuscate(buffer2, fontKey));
+    for (const [i, { data: buffer2, fontKey }] of file.FontTable.fontOptionsWithKey.entries()) {
+      zip.file(`word/fonts/font${i + 1}.odttf`, obfuscate(buffer2, fontKey));
     }
     return zip;
   }
diff --git a/dist/index.umd.cjs b/dist/index.umd.cjs
index d36d7606f4b7e7e93a48c30b5060d5a96a37236c..12ff124c1a765fd5f75b27a0130b2d8478a58eed 100644
--- a/dist/index.umd.cjs
+++ b/dist/index.umd.cjs
@@ -14081,7 +14081,7 @@ var __async = (__this, __arguments, generator) => {
         this.relationships.addRelationship(
           i + 1,
           "http://schemas.openxmlformats.org/officeDocument/2006/relationships/font",
-          `fonts/${options[i].name}.odttf`
+          `fonts/font${i + 1}.odttf`
         );
       }
     }
@@ -21601,9 +21601,8 @@ var __async = (__this, __arguments, generator) => {
           zip.file(`word/media/${data.fallback.fileName}`, data.fallback.data);
         }
       }
-      for (const { data: buffer2, name, fontKey } of file.FontTable.fontOptionsWithKey) {
-        const [nameWithoutExtension] = name.split(".");
-        zip.file(`word/fonts/${nameWithoutExtension}.odttf`, obfuscate(buffer2, fontKey));
+      for (const [i, { data: buffer2, fontKey }] of file.FontTable.fontOptionsWithKey.entries()) {
+        zip.file(`word/fonts/font${i + 1}.odttf`, obfuscate(buffer2, fontKey));
       }
       return zip;
     }
